src/main/java/net/mrscauthd/beyond_earth/events/Methods.java

```diff
@@ -89,72 +99,66 @@ public static void worldTeleport(Player entity, ResourceKey<Level> planet, doubl
         }
     }
 
-    public static boolean netheriteSpaceSuitCheck(LivingEntity entity) {
-        Boolean item3 = checkArmor(entity, 3, ModInit.NETHERITE_OXYGEN_MASK.get());
-        Boolean item2 = checkArmor(entity, 2, ModInit.NETHERITE_SPACE_SUIT.get());
-        Boolean item1 = checkArmor(entity, 1, ModInit.NETHERITE_SPACE_PANTS.get());
-        Boolean item0 = checkArmor(entity, 0, ModInit.NETHERITE_SPACE_BOOTS.get());
+    /**
+     * @param entity the LivingEntity to check
+     * @return true if entity is wearing a full set of netherite space gear, false otherwise
+     */
+    public static boolean netheriteSpaceSuitCheck(final LivingEntity entity) {
+        if (!checkArmor(entity, 3, ModInit.NETHERITE_OXYGEN_MASK.get())) return false; // missing a netherite oxygen mask
+        if (!checkArmor(entity, 2, ModInit.NETHERITE_SPACE_SUIT.get())) return false; // missing a netherite space suit
+        if (!checkArmor(entity, 1, ModInit.NETHERITE_SPACE_PANTS.get())) return false; // missing netherite space pants
+        if (!checkArmor(entity, 0, ModInit.NETHERITE_SPACE_BOOTS.get())) return false; // missing netherite space boots
 
-        if (item0 && item1 && item2 && item3) {
-            return true;
-        }
-        return false;
+        return true; // has mask + suit + pants + boots
     }
 
-    public static boolean spaceSuitCheck(LivingEntity entity) {
-        Boolean item3 = checkArmor(entity, 3, ModInit.OXYGEN_MASK.get());
-        Boolean item2 = checkArmor(entity, 2, ModInit.SPACE_SUIT.get());
-        Boolean item1 = checkArmor(entity, 1, ModInit.SPACE_PANTS.get());
-        Boolean item0 = checkArmor(entity, 0, ModInit.SPACE_BOOTS.get());
+    /**
+     * @param entity the LivingEntity to check
+     * @return true if entity is wearing a full set of standard space gear, false otherwise
+     */
+    public static boolean spaceSuitCheck(final LivingEntity entity) {
+        if (!checkArmor(entity, 3, ModInit.OXYGEN_MASK.get())) return false;
+        if (!checkArmor(entity, 2, ModInit.SPACE_SUIT.get())) return false;
```

Comment:
Return false early if we know the player isn't wearing a part of the space suit. This avoids checking all parts of the space suit if we know one of them is missing.